ibm/service/iamaccessgroup/resource_ibm_iam_access_group_members.go

```diff
@@ -164,6 +164,10 @@ func resourceIBMIAMAccessGroupMembersRead(context context.Context, d *schema.Res
 	listAccessGroupMembersOptions.SetLimit(limit)
 	members, detailedResponse, err := iamAccessGroupsClient.ListAccessGroupMembers(listAccessGroupMembersOptions)
 	if err != nil {
+		if detailedResponse != nil && detailedResponse.StatusCode == 404 {
```

Comment:
Does the list gives 404 when the list is empty?

Reply:
yes it gives 404 when access group itself is deleted